src/main/java/de/teamlapen/vampirism/VampirismMod.java

```diff
@@ -87,13 +87,13 @@ public ItemStack createIcon() {
      * {@link VReference} instead of this one. This is only here to init it as early
      * as possible
      */
-    private final static EntityClassification HUNTER_CREATURE_TYPE = EntityClassification.create("vampirism_hunter", "test1", 25, false, false); //TODO 1.14 use debugger to find out what test1 does
+    private final static EntityClassification HUNTER_CREATURE_TYPE = EntityClassification.create("vampirism_hunter", "vampirism_hunter", 25, false, false);
```

Comment:
first string is the object name and second is the name of the entityclassification (see net.minecraftforge.common.IExtensibleEnum), don't know why they use an extra attribute for the name